roles/ovirt-host-deploy-firewalld/vars/firewalld_rules_common.yml

```diff
@@ -0,0 +1,9 @@
+host_deploy_firewalld_rules:
+  ## Service definition:
+  - service: rpc-bind
+  - service: snmp
+  - service: cockpit
+
+  ## Port definition:
+  # ovirt-imageio-daemon
+  - port: 54322/tcp
```

Comment:
I can't find it. According to spec file, there is no:

 https://github.com/oVirt/ovirt-imageio/blob/master/daemon/ovirt-imageio-daemon.spec.in#L52